cmd/createCPTVFile/main.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 The Cacophony Project
```

Comment:
Can you call the package (and resulting binary) something shorter like `createcptv`